jenkins.check_copyright.sh

```diff
@@ -32,7 +32,7 @@ git diff --name-only `git merge-base origin/master HEAD` HEAD |
     grep -v -E '\.def$' |
     grep -v -E '\.inc$' |
     grep -v -E 'test/benchmarks/.*\.js$' |
-    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; if [ \$? -ne 0 ]; then cat $ERRFILETEMP >> $ERRFILE; fi"
+    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; || cat $ERRFILETEMP >> $ERRFILE; fi"
```

Comment:
@goyakin @Francososa it appears it did complain but the logic of the script was such that it did not cause the check to fail.

http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/ubuntu_check_copyright_prtest/619/console

```
20:54:13 sh: 1: Syntax error: "||" unexpected
20:54:13 sh: 1: Syntax error: "||" unexpected
```

I'll make sure to double check the console output on the next iteration.